### ISSUE TITLE
Three buttons on /Home have no discernible text

### ISSUE THREAD

**Issue by youbrokesomething** [Automattic/wp-calypso#75592]
### Quick summary

Three buttons on /home do not have discernible text or labels that would give context to screen readers.
The buttons are:
`.navigation-link.button[type="button"]:nth-child(1)`
`.navigation-link.button[type="button"]:nth-child(3)`
and `.domain-upsell__card-dismiss > button`.

The buttons should have either an accessible name or a label to give context.

### Steps to reproduce

1. On /home.
2. Run an accessibility check using a tool of your choice (I prefer axe DevTools Chrome extension) or inspect the buttons specified above.

### What you expected to happen

All buttons should have discernible names that provide context to devices like screen readers.

### What actually happened

Three buttons have no name.

### Impact

Some (< 50%)

### Available workarounds?

No but the platform is still usable

**Comment by davemart-in** [comment]
PR: https://github.com/Automattic/wp-calypso/pull/76951

**Comment by davemart-in** [comment]
Should be fixed now.
